Frends.MySQL.ExecuteProcedure/Frends.MySQL.ExecuteProcedure.Tests/UnitTests.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Frends.MySQL.ExecuteProcedure.Definitions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MySql.Data.MySqlClient;
+
+namespace Frends.MySQL.ExecuteProcedure.Tests;
+
+public class UnitTests
+{
+    [TestClass]
+    public class MySqlQueryTests
+    {
+        private readonly string _server = Environment.GetEnvironmentVariable("MySQL_server");
+        private readonly string _uid = Environment.GetEnvironmentVariable("MySQL_uid");
+        private readonly string _pwd = Environment.GetEnvironmentVariable("MySQL_pwd");
+        private readonly string _database = Environment.GetEnvironmentVariable("MySQL_database");
```

Comment:
You should instruct the developer to use docker container instead of real server. Then you don't have to get these values from environment variables, but you can hardcode them to the tests. Example: https://github.com/FrendsPlatform/Frends.PostgreSQL/blob/main/Frends.PostgreSQL.ExecuteQuery/Frends.PostgreSQL.ExecuteQuery.Tests/ExecuteQueryTests.cs#L21